frontend/src/scenes/insights/views/WorldMap/WorldMap.scss

```diff
@@ -10,7 +10,8 @@
         stroke-width: 0.125rem;
 
         &:hover {
-            color: var(--primary-3000-hover) !important;
+            color: var(--primary) !important;
+            opacity: 0.9;
```

Comment:
WDYT @thmsobrmlr?

Reply:
This was actually the case before, I think the change there was somewhat inadvertent https://github.com/PostHog/posthog/commit/0bbd150e7aa16ebf971e0c64f865c47c494e0e5a#diff-d78faac74e21292285299e001cbefb346090564c7fa31db0e4ba8b496d9ae1d7L13-R13

Reply:
I had actually tried `--primary-dark` before and decided against it, because it subjectively didn't feel right. Giving this another go, maybe just `opacity: 0.8` works?

![2023-11-28 15 45 15](https://github.com/PostHog/posthog/assets/1851359/65398040-eb27-42bf-b2be-13313bc64b32)

Reply:
I've found a (darker) color that I'm happy with. Check it out and let me know what you think @Twixes.